.github/workflows/artifacts.yaml

```diff
@@ -176,19 +176,29 @@ jobs:
       # the correct location that we extracted the archive. This seems more reliable
       # than using `--define-prefix` - it seems to tack an extra 'lib/' subcomponent
       # onto the include path that breaks the build.
-      - name: Fix pkg-config prefix
-        # We use bash shell explicitly to avoid PowerShell on Windows and to ensure we have 'sed'.
+      - name: Fix pkg-config prefix (UNIX)
+        if: matrix.os != 'windows-latest'
```

Comment:
nit: I think `if: runner.os != 'Windows'` is a more direct expression of intent, in this hellhole of yaml and godawful nonsense

Reply:
Good call. I fixed this in a separate commit because it turns out to be a pattern that I was using in the `test.yaml` and `daily-tests.yaml` workflows as well. I've switched all instances to use `runner.os` instead.